lib/sigsci-agent.sh

```diff
@@ -106,7 +106,7 @@ then
     echo "-----> Downloading sigsci-agent"
     curl -s --retry 45 --retry-delay 2 -o sigsci-agent_${SIGSCI_AGENT_VERSION}.tar.gz https://dl.signalsciences.net/sigsci-agent/${SIGSCI_AGENT_VERSION}/linux/sigsci-agent_${SIGSCI_AGENT_VERSION}.tar.gz > /dev/null
 
-    if [ -n $SIGSCI_CHECKSUM_INTEGRITY_CHECK ]
+    if [ -z $NO_SIGSCI_CHECKSUM_INTEGRITY_CHECK ]
```

Comment:
If you make it:
`SIGSCI_DISABLE_CHECKSUM_INTEGRITY_CHECK`
it will show up alphabetically with all our environment variables, rather than out of order if there are many variables for the app.